### PR TITLE
Add deon

### PR DESCRIPTION
**Pull request by jayqi**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jayqi** [comment]
@pjbull @ejm714 please confirm in a comment here that you are willing to be listed as a maintainer for this recipe.

**Comment by pjbull** [comment]
Confirming I will be a maintainer!

File: recipes/deon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    data science, machine learning, and AI is increasingly important. The goal
+    of deon is to push that conversation forward and provide concrete,
+    actionable reminders to the developers that have influence over how data
+    science gets done.
+  doc_url: http://deon.drivendata.org/
+  dev_url: https://github.com/drivendataorg/deon
+
+extra:
+  recipe-maintainers:
+    - ejm714
```
Comment (chrisburr) [inline recipes/deon/meta.yaml]: @ejm714 Can you comment about if you're happy to be listed as a maintainer here?

Comment (ejm714) [reply recipes/deon/meta.yaml]: Yep happy to be a maintainer!

**Pull request merged**